controllers/activemqartemis_controller.go

```diff
@@ -143,6 +146,8 @@ func (r *ActiveMQArtemisReconciler) Reconcile(ctx context.Context, request ctrl.
 
 		UpdatePodStatus(customResource, r.Client, request.NamespacedName)
 
+		err = UpdateCRStatus(customResource, r.Client, request.NamespacedName)
```

Comment:
I think the retry on error logic is broken because it used a 10h default retry period (in error). That may need to be fixed to make any retry on status error visible in a sensible time frame.

Reply:
30s is better. thanks!